src/allshoes.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./App.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircle } from "@fortawesome/free-solid-svg-icons";
+import { Slide } from "react-awesome-reveal";
+
+export default function Allshoes(props) {
+  const { AllshoesName, AllImage } = props;
+  return (
+    <>
+      <Slide triggerOnce className="slide">
+        <div className="container">
+          <div className="listContainer">
+            {AllshoesName.map((ele) => {
+              return (
+                <div className="infoCard">
```

Comment:
Again, needs a `key` prop

```suggestion
                <div className="infoCard" key={ele.id}>
```